### PR TITLE
Migrate microbenchmark build files to kts

### PR DESCRIPTION
**Pull request by keyboardsurfer**

### CODE REVIEW

File: MicrobenchmarkSample/app/build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,46 @@
+plugins {
+    id("com.android.application")
+    id("org.jetbrains.kotlin.android")
```
Comment (mlykotom) [inline MicrobenchmarkSample/app/build.gradle.kts:3]:
```suggestion
    id("kotlin-android")
```

File: MicrobenchmarkSample/benchmarkable/build.gradle [hunk 2]
```diff
@@ -1,31 +1,31 @@
-apply plugin: 'com.android.library'
-apply plugin: 'kotlin-android'
+apply plugin: "com.android.library"
```
Comment (mlykotom) [inline MicrobenchmarkSample/benchmarkable/build.gradle:1]: can we convert this file to kts as well?

File: MicrobenchmarkSample/gradle/libs.versions.toml [hunk 3]
```diff
@@ -0,0 +1,40 @@
+[versions]
+agp = "7.3.0"
```
Comment (mlykotom) [inline MicrobenchmarkSample/gradle/libs.versions.toml:2]:
```suggestion
agp = "7.3.1"
```

**Review by mlykotom — APPROVED** [review]

File: MicrobenchmarkSample/app/build.gradle.kts (hunk 1 above)
Comment (keyboardsurfer) [reply MicrobenchmarkSample/app/build.gradle.kts:3]: Done

File: MicrobenchmarkSample/gradle/libs.versions.toml (hunk 3 above)
Comment (keyboardsurfer) [reply MicrobenchmarkSample/gradle/libs.versions.toml:2]: Done

File: MicrobenchmarkSample/benchmarkable/build.gradle (hunk 2 above)
Comment (keyboardsurfer) [reply MicrobenchmarkSample/benchmarkable/build.gradle]: Oops. done

**Pull request merged**